streams/src/main/java/org/apache/kafka/streams/processor/TaskId.java

```diff
@@ -25,11 +25,13 @@
 import java.nio.ByteBuffer;
 
 /**
- * The task id representation composed as topic group id plus the assigned partition id.
+ * The task ID representation composed as topic group ID plus the assigned partition ID.
```

Comment:
Nitpick:

> Represents the task id as a composition of topic group id and the assigned partition id.

(This change also implies that we do not commit on a contract of how this composition is implemented, i.e. we leave it open whether it's topic-group-id followed by assigned partition id, or vice versa, or...)